src/App.tsx

```diff
@@ -1,26 +1,131 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { deleteTodo, getTodos, createTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+import { TodoCompletedCategory } from './types/TodoCompletedCategory';
+import { filterTodosByComplated } from './utils/filterTodosByCompleted';
+import { Errors } from './types/Errors';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [completedCategory, setCompletedCategory] =
+    useState<TodoCompletedCategory>(TodoCompletedCategory.all);
+  const [errorMessage, setErrorMessage] = useState<Errors>(Errors.noneError);
+  const [removedTodosId, setRemovedTodosId] = useState([0]);
+
+  const filtredTodos = filterTodosByComplated(todos, completedCategory);
+  const countOfCompletedTodos = todos.filter(todo => todo.completed).length;
+  const countOfNotCompletedTodos = todos.length - countOfCompletedTodos;
+  const isSomeTodoComplated = todos.some(todo => todo.completed);
+
+  async function fetchTodosFromApi() {
+    try {
+      const todosFromApi = await getTodos();
+
+      setTodos(todosFromApi);
+    } catch {
+      setErrorMessage(Errors.loadError);
+    }
+  }
+
+  async function fetchAddTodo(newTitle: string) {
```

Comment:
Would be a good idea to rename `fetchAddTodo `for clarity and consistency. Since it involves adding a new todo, you could consider using the `handle `prefix instead.